libmscore/style.cpp

```diff
@@ -533,7 +533,7 @@ static const StyleType styleTypes[] {
       { Sid::defaultFrameRound,             "defaultFrameRound",             0 },
       { Sid::defaultFrameFgColor,           "defaultFrameFgColor",           QColor(0, 0, 0, 255) },
       { Sid::defaultFrameBgColor,           "defaultFrameBgColor",           QColor(255, 255, 255, 0) },
-      { Sid::defaultOffset,                 "defaultOffset",                 QPointF() },
+      { Sid::defaultOffset,                 "defaultOffset",                 QPointF(0.0,0.5) },
```

Comment:
Here a space after the comma is missing

Reply:
Actually why changung default offset at all? It seems to scrwe up the mtest?

Reply:
Actually why changing default offset at all? It sems to be causing the mtest failures Travis CI reports

Reply:
Agreed, this seems wrong.  If the way we have some properties inherit from the default makes the main change to footerOffset insufficient, then we need to change where that inheritance happens.  But the one-line changes to footerOffset *should have been sufficient.

Reply:
@MarcSabatella So, what you are saying is that it is sufficient to add the footer offset value?

Reply:
> Actually why changing default offset at all? It sems to be causing the mtest failures that Travis CI reports
Because in the issue tracker it was mentioned that it was the case with all the scores so I though why not add this value to the default values being provided.

Reply:
> Here a space after the comma is missing

Sir, I have added it my 2nd commit. and Now I have Squashed those commits.

Reply:
Still the same issue, the missing space and the superfluous defaultoffset change

Reply:
I don’t know why its showing like that. I will look into it asap.